typescript/cosmos-types/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@hyperlane-xyz/cosmos-types",
+  "version": "1.0.0",
+  "description": "Hyperlane TypeScript SDK types for the Cosmos Hyperlane SDK module",
+  "type": "module",
+  "exports": {
+    ".": "./dist/index.js"
+  },
+  "types": "dist/index.d.ts",
+  "files": [
+    "/dist"
+  ],
+  "homepage": "https://www.hyperlane.xyz",
+  "repository": "https://github.com/hyperlane-xyz/hyperlane-monorepo",
+  "keywords": [
+    "hyperlane",
+    "cosmos",
+    "blockchain",
+    "sdk",
+    "types"
+  ],
+  "license": "Apache-2.0",
+  "scripts": {
+    "build": "rimraf dist && tsc",
+    "format": "prettier --write .",
+    "lint": "eslint -c ./eslint.config.mjs",
+    "prettier": "prettier --write ./src",
+    "clean": "rm -rf ./dist ./cache",
+    "test": "exit 0"
+  },
+  "devDependencies": {
+    "@eslint/js": "^9.15.0",
+    "@typescript-eslint/eslint-plugin": "^8.1.6",
+    "@typescript-eslint/parser": "^8.1.6",
+    "eslint": "^9.15.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-import-resolver-typescript": "^3.6.3",
+    "eslint-plugin-import": "^2.31.0",
+    "globals": "^15.14.0",
+    "prettier": "^2.8.8",
+    "rimraf": "^6.0.1",
+    "ts-node": "^10.8.0",
+    "tslib": "^2.8.1",
+    "typescript": "5.3.3",
+    "typescript-eslint": "^8.23.0"
```

Comment:
do we need this if we have the other two?

Reply:
removed unused dependencies